app/main.development.js

```diff
@@ -38,8 +63,52 @@ const installExtensions = async () => {
   }
 };
 
+const launchDCRd = async() => {
+  var spawn = require('child_process').spawn;
+  var args = ['--piperx=3'];
+  if (mainNet == false) {
+     args.push('--testnet');
+  }
+  console.log(`starting dcrd with ${args}`);
+  var r = spawn('dcrd', args, { detached: true, stdio: [ 'ignore', 'ignore', 'ignore', 'pipe' ] });
+  r.unref();
+
+  r.on('close', (code) => {
+    console.log(`child process exited with code ${code}`);
+  });
+
+  console.log('end of launch DCRd');
+}
+
+const launchDCRWallet = async () => {
+  var spawn = require('child_process').spawn;
+  var args = ['--noinitialload','--tlscurve=P-256','--onetimetlskey','--piperx=4'];
+  args.push('--appdata',appDataDirectory());
+  args.push('--experimentalrpclisten','127.0.0.1:' + RPCWalletPort());
+  if (mainNet == false) {
+     args.push('--testnet');
+  }
+  console.log(`starting dcrwallet with ${args}`);
+  var r = spawn('dcrwallet', args, { detached: true, stdio: [ 'ignore', 'inherit', 'inherit', 'ignore', 'pipe'  ] });
+  r.unref();
+
+  r.stdout.on('data', (data) => {
+    console.log(`stdout: ${data}`);
+  });
+
+  r.stderr.on('data', (data) => {
+    console.log(`stderr: ${data}`);
+  });
+
+  r.on('close', (code) => {
+    console.log(`child process exited with code ${code}`);
+  });
+};
+
 app.on('ready', async () => {
   await installExtensions();
+  await launchDCRd();
```

Comment:
Should probably either be launchDCRD or make the wallet one launchDCRwallet.  Either way, just consistent.